content/developer/rom/dossier/respondents.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Respondents
+status: draft
+---
+
+* TOC
+{:toc}
+
+Respondents are people that fill out questionnaires. The default respondent is the patient, but there's also profess, parent, second_parent, teacher and caregiver.
+
+- In the future second_parent will be merged into parent.
+- You won't see a profess respondent, since you can't send out to professionals, since there isn't usually just one.
+
+
+## Get a single respondent
+
+    GET /api/v1/dossier/:epd_id/respondents/:id
```

Comment:
Heeft deze endpoint wel zin als je op ID moet opvragen? In de docs kan ik geen enkele plek vinden waar respondent_id gereturnd wordt. Dan zal niemand behalve wij hem kunnen weten. Misschien zinniger om in plaats van `:id` een `:respondent_type` param te hebben.